src/main/resources/org/jenkinsci/plugins/xunit/types/boosttest-1.2-to-junit-4.xsl

```diff
@@ -217,7 +219,10 @@ THE SOFTWARE.
 
 
             <xsl:attribute name="time">
-                <xsl:value-of select="$time div 1000000"/>
+                <xsl:choose>
+                    <xsl:when test="TestingTime"><xsl:value-of select="$time div 1000000"/></xsl:when>
+                    <xsl:otherwise>0</xsl:otherwise>
```

Comment:
Time should be 0.000

Reply:
No problem. I'll push a patch addressing this in just a bit